### PR TITLE
Additional integration of provider into the image 

### PR DESCRIPTION
**Pull request by leason00**
Signed-off-by: lisen <lisen@youzan.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,10 @@ COPY ossutil /usr/bin
 
 ENV RETRIEVE_TF_PROVIDER=/tmp/retrieve_tf_provider.sh
 
-RUN $RETRIEVE_TF_PROVIDER random 3.1.0
-RUN $RETRIEVE_TF_PROVIDER alicloud 1.140.0
+RUN $RETRIEVE_TF_PROVIDER hashicorp random 3.1.0
+RUN $RETRIEVE_TF_PROVIDER aliyun alicloud 1.140.0
```
Comment (zzxwill) [inline Dockerfile]:
```suggestion
RUN $RETRIEVE_TF_PROVIDER hashicorp alicloud 1.140.0
```

File: Dockerfile [hunk 2]
```diff
@@ -23,7 +23,10 @@ COPY ossutil /usr/bin
...
+RUN $RETRIEVE_TF_PROVIDER tencentcloudstack tencentcloud 1.72.0
 
-ENTRYPOINT ["tail", "-f", "/dev/null"]
+RUN cp -r .terraform.d /root/.terraform.d
```
Comment (zzxwill) [inline Dockerfile]: No need any more.

**Comment by leason00** [comment]
@zzxwill

**Review by zzxwill — APPROVED** [review]
Great job!

**Review by wonderflow — APPROVED** [review]

**Pull request merged**